content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,208 @@
+===================
+Fleet configuration
+===================
```

Comment:
What about "configuration" only? it is in the "Fleet" category. But see also my suggestion in the main comment.

Reply:
Hi @jcs-odoo - I have two documents currently written in github for fleet, and I am working on a third. So ultimately, they will all be nested under 'fleet', so I didn't want to make the default fleet doc be this one. 

I had done the format you suggested for other apps, where main doc I wrote covered 90% of things, but this time I was trying to chop the docs up into smaller digestible pieces, so that's why I tried to make separate docs for different items (addling a  vehicle, configurations, reports, repairs, etc).  If you don't think that's appropriate or a good idea, I can combine all the docs into one, but I was hoping to try and make the docs less overwhelming by having them separated out. Does that sound good?

I also know it means that once one doc is approved and posted, I will need to change the TOC so it looks correct with the newly added docs. I didn't want to wait to have the first doc be posted for review, have it go live, then add the rest, one at a time. That is why there are 2 (with another one on the way) posted now for review.  Sound good?

Reply:
Hi @larm-odoo 
You're right, it's better to have several pull requests rather than a massive one, at least in this documentation repository. Once the first PR is merged, you'll probably have a conflict on the other ones until you fix the toc tree. So that's fine :)

I agree this content only wouldn't make that much sense for the main page. Now that I see it again, I think my suggestion isn't better. "Configuration" alone is too vague (a bit like "misc." or "advanced"). Maybe "vehicle models and manufacturers"? That would be more representative. It's up to you :) 

Have a good day